examples/simple_real_numbers.jl

```diff
@@ -55,6 +58,59 @@ function simple_real_numbers(context)
 
     result_sv_shift2 = decrypt(sv_shift2, private_key)
     println("x1 shifted circularly by 2 = ", result_sv_shift2)
+
+    ################################################################################
+    init_matrix_rotation!(context, private_key, [(1, 0), (0, 1)], (2, 3))
+
+    m1 = [0.25 0.5 0.75; 1.0 2.0 3.0]
+    m2 = [3.0 2.0 1.0; 0.75 0.5 0.25]
```

Comment:
```suggestion
    m1 = [0.25 0.50 0.75;
          1.00 2.00 3.00]
    m2 = [3.00 2.00 1.00;
          0.75 0.50 0.25]
```
Maybe for increased readability?